src/forms/redux-form/textarea.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import ReduxFormTextarea from './Textarea';
+
+describe('redux-form Textarea', () => {
+
+	const formAttrs = {
+		input: {
```

Comment:
nitpick: maybe make these attributes more suitable for a textarea? 
maxLength 20 is short.